packages/blocks/src/_common/utils/types.ts

```diff
@@ -137,9 +137,10 @@ export type Selectable = EdgelessElement;
 
 export type Erasable = EdgelessElement;
 
-export type Connectable =
-  | Exclude<TopLevelBlockModel, FrameBlockModel>
-  | Exclude<PhasorElement, ConnectorElement | BrushElement>;
+export type Connectable = Exclude<
```

Comment:
This maybe includes `TopLevelBlockModel`?

Reply:
I will add it back. Thanks!